src/main/java/core/controller/driver/DeleteDriverController.java

```diff
@@ -0,0 +1,24 @@
+package core.controller.driver;
+
+import core.lib.Injector;
+import core.service.driver.DriverService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class DeleteDriverController extends HttpServlet {
+    private static final Injector injector
+            = Injector.getInstance("core");
+    private final DriverService driverService
+            = (DriverService) injector.getInstance(DriverService.class);
+    
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String id = req.getParameter("id");
+        driverService.delete(Long.valueOf(id));
+        resp.sendRedirect(req.getContextPath() + "/drivers/all");
```

Comment:
```suggestion
        resp.sendRedirect(req.getContextPath() + "/drivers/");
```

Reply:
Hmmm... i have been chacking all the url few times, still missed this one. Thanks